aws_policies_cis/aws_cloudtrail_s3_bucket_access_logging.py

```diff
@@ -1,5 +1,8 @@
+from panther import resource_lookup  # pylint disable:import-error
+
+
 def policy(resource):
-    # TODO: Use the get_resource() helper func here
-    return (resource['Bucket']['LoggingPolicy'] is not None or
-            # Verify that the bucket exists in the same account
-            resource['Bucket']['Location'] is None)
+    bucket_arn = 'arn:aws:s3:::' + resource['S3BucketName']
+    bucket = resource_lookup(bucket_arn)
```

Comment:
How does resource_lookup behave if there's nothing found?

Reply:
It raises an exception. I thought about catching the exception, but then users wouldn't know if there bucket was messed up. For example, we have a cloudtrail raising an exception here because it's logging to a bucket that no longer exists! Not sure if that info should just be tossed away. Although I suppose that could be a different policy...